docs/specs/clients/notify/error-codes.md

```diff
@@ -1,27 +1,7 @@
 # Error Codes
 
-## INVALID
+## Signing
 
 ```sh
-case .invalidProposal: return 1000
-```
-
-## REJECTED
-
-```sh
-case .userRejected return 5000
-```
-
-## REASON
-
-```sh
-case .userUnsubscribed return 6000
-case .userHasExistingSubscription return 6001
-```
-
-## FAILURE
-
-```sh
-case .approvalFailed: return 7002
```

Comment:
We don't want these errors anymore?

Reply:
I think we can remove `approvalFailed` and `rejectionFailed` and assumed others are also not valid anymore. Will remove those two only

Reply:
also `invalidProposal` is no longer valid

Reply:
Makes sense to me

Reply:
Brought back `userUnsubscribed` and `userHasExistingSubscription`

Reply:
okay good i thought i was missing something :sob: